_wormtablemodule.c

```diff
@@ -22,16 +22,25 @@
 #include <db.h>
 #include "halffloat.h"
 
+/* $VR$: Dealing with Windows... */
```

Comment:
I think we can delete this line --- adding author stuff just clutters things I think, that's what git is for!